### PR TITLE
Update to outdated files in the dev-1.16-ko.7

### PR DESCRIPTION
**Pull request by ysyukr**
### 24 out of 24 files to be modified
- [x] content/ko/_index.html
- [x] content/ko/docs/concepts/architecture/cloud-controller.md
- [x] content/ko/docs/concepts/overview/what-is-kubernetes.md
- [x] content/ko/docs/concepts/workloads/controllers/daemonset.md
- [x] content/ko/docs/concepts/workloads/controllers/statefulset.md
- [x] content/ko/docs/concepts/workloads/controllers/ttlafterfinished.md
- [x] content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md
- [x] content/ko/docs/reference/glossary/cloud-provider.md
- [x] content/ko/docs/reference/glossary/etcd.md
- [x] content/ko/docs/reference/kubectl/cheatsheet.md
- [x] content/ko/docs/reference/using-api/client-libraries.md
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/best-practices/certificates.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/container-runtimes.md
- [x] content/ko/docs/setup/production-environment/tools/kops.md
- [x] content/ko/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md
- [x] content/ko/docs/tasks/access-application-cluster/configure-dns-cluster.md
- [x] content/ko/docs/tasks/administer-cluster/cluster-management.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [x] content/ko/docs//tasks/run-application/horizontal-pod-autoscale.md
- [x] content/ko/docs/tasks/tools/install-minikube.md
- [x] content/ko/docs/tutorials/configuration/configure-redis-using-configmap.md

from #18048
/language ko

### CODE REVIEW

File: content/ko/docs/reference/glossary/cloud-provider.md [hunk 1]
```diff
@@ -4,14 +4,28 @@ id: cloud-provider
...
 - community
 ---
- 클라우드 공급자는 쿠버네티스 클러스터를 실행할 수 있는 클라우드 컴퓨팅 플랫폼을 제공하는 회사.
+ 클라우드 컴퓨팅 플랫폼을 제공하는 사업자 또는 다른 조직
 
-<!--more--> 
+<!--more-->
 
-클라우드 컴퓨팅 플랫폼을 제공하는 클라우드 공급자 또는 클라우드 서비스 공급자(CSP)라고 한다. 이들은 인프라 서비스(IaaS) 또는 플랫폼 서비스(PaaS)를 제공할 수 있다. 클라우드 공급자는 쿠버네티스 클러스터를 호스트 하며 클러스터와 상호작용하는 로드밸런서, 스토리지 클래스 등의 서비스도 제공한다.
+클라우드 공급자, 때로는 클라우드 서비스 공급자(CSP) 부르며,
```
Comment (seokho-son) [inline content/ko/docs/reference/glossary/cloud-provider.md]:
```suggestion
클라우드 공급자, 때로는 클라우드 서비스 공급자(CSP)라 부르며,
```

File: content/ko/docs/reference/glossary/cloud-provider.md [hunk 2]
```diff
@@ -4,14 +4,28 @@ id: cloud-provider
...
+  클라우드 컴퓨팅 플랫폼을 제공하는 조직.
 
-aka: 
+aka:
+- Cloud Service Provide
 tags:
 - community
 ---
- 클라우드 공급자는 쿠버네티스 클러스터를 실행할 수 있는 클라우드 컴퓨팅 플랫폼을 제공하는 회사.
+ 클라우드 컴퓨팅 플랫폼을 제공하는 사업자 또는 다른 조직
```
Comment (seokho-son) [inline content/ko/docs/reference/glossary/cloud-provider.md]:
```suggestion
 클라우드 컴퓨팅 플랫폼을 제공하는 사업자 또는 다른 조직.
```

File: content/ko/docs/reference/glossary/cloud-provider.md [hunk 3]
```diff
@@ -4,14 +4,28 @@ id: cloud-provider
...
+클라우드 공급자, 때로는 클라우드 서비스 공급자(CSP) 부르며,
+클라우드 컴퓨팅 플랫폼 또는 서비스를 제공한다.
+
+많은 클라우드 공급자들은 관리되는 인프라를 제공한다(이를
+Infrastructure as a Service 또는 IaaS 라 부른다).
+관리되는 인프라를 통해 클라우드 공급자는
+서버, 스토리지 그리고 네트워킹을 담당하고 쿠버네티스
+클러스터 실행과 같은 계층을 관리한다.
+
+사용자는 쿠버네티스를 관리되는 서비스로 찾을 수 있다. 때로는 이것을
```
Comment (seokho-son) [inline content/ko/docs/reference/glossary/cloud-provider.md]:
```suggestion
사용자는 쿠버네티스를 관리되는 서비스(managed service)로도 찾을 수 있다. 때로는 이것을
```
서비스가 관리되는 것으로 오해할 소지가 있을 것 같습니다.
managed service 가 서비스의 타입이기도 하고, 메니지드 서비스 로 읽기도 해서 번역을 하기에 다소 난해한 측면이 있네요. 일단 한영 병기를 하면 오해를 줄일 수 있을 것 같고, 필요하다면 메니지드(managed) 서비스 와 같은 형태로 번역하면 어떨까 싶습니다. 컨트리뷰터의 선호에 따라 진행하시면 어떨까 합니다. :)

File: content/ko/docs/setup/production-environment/tools/kops.md [hunk 4]
```diff
@@ -39,22 +39,82 @@ kops를 이용하기 위해서는 [kubectl](/docs/tasks/tools/install-kubectl/)
...
+
+특정 버전의 명령 부분이다.
+
+예를 들어 kops 버전을 v1.15.0을 다운로드 하려면 다음을 입력한다.
+
+```shell
+curl -LO  https://github.com/kubernetes/kops/releases/download/1.15.0/kops-darwin-amd64
+```
+
+kops 바이너리를 실행가능하게 만든다.
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/tools/kops.md]:
```suggestion
kops 바이너리를 실행 가능하게 만든다.
```

File: content/ko/docs/setup/production-environment/tools/kops.md [hunk 5]
```diff
@@ -39,22 +39,82 @@ kops를 이용하기 위해서는 [kubectl](/docs/tasks/tools/install-kubectl/)
...
+```
+특정 버전의 명령 부분이다.
+
+예를 들어 kops 버전을 v1.15.0을 다운로드 하려면 다음을 입력한다.
+
+```shell
+curl -LO  https://github.com/kubernetes/kops/releases/download/1.15.0/kops-linux-amd64
+```
+
+kops 바이너리를 실행가능하게 만든다.
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/tools/kops.md]:
```suggestion
kops 바이너리를 실행 가능하게 만든다.
```

**Comment by ysyukr** [comment]
@seokho-son 리뷰 감사합니다. 주신 제안은 확인해서 반영했습니다 :)

**Comment by seokho-son** [comment]
@ysyukr Thank you !
/lgtm
/approve

**Pull request merged**